src/registrar/models/domain.py

```diff
@@ -139,6 +139,33 @@ class State(models.TextChoices):
         # previously existed but has been deleted from the registry
         DELETED = "deleted", "Deleted"
 
+        @classmethod
+        def get_help_text(cls, state) -> str:
```

Comment:
Is there a better architectural way to do this? Im not really a huge fan of this solution